src/Compilers/CSharp/Test/Emit/PDB/PDBUsingTests.cs

```diff
@@ -569,7 +569,7 @@ class A { void M() {  } }
 ");
         }
 
-        [Fact]
+        [ConditionalFact(typeof(NoUsedAssembliesValidation))] // Tracked by https://github.com/dotnet/roslyn/issues/60045
```

Comment:
Why do you use comments here vs. putting the URL into the `Reason` property?

Reply:
Not much reason. Either one works

Reply:
One appears in the test logs, one does not 🙂.